Resources/Prototypes/Entities/Clothing/OuterClothing/vests.yml

```
@@ -21,9 +21,9 @@
 
 #Mercenary web vest
 - type: entity
-  parent: [ClothingOuterVestWeb, BaseMinorContraband] #web vest so it should have some pockets for ammo
+  parent: [ClothingOuterStorageBase, AllowSuitStorageClothing,  BaseMajorContraband] #web vest so it should have some pockets for ammo
```

```suggestion
  parent: [ BaseMajorContraband, ClothingOuterVestWeb] #web vest so it should have some pockets for ammo
```
you can do it like this instead of reparenting